src/compiler/compile/render-dom/wrappers/Title.ts

```diff
@@ -14,13 +14,13 @@ export default class TitleWrapper extends Wrapper {
 		block: Block,
 		parent: Wrapper,
 		node: Title,
-		strip_whitespace: boolean,
-		next_sibling: Wrapper
+		_strip_whitespace: boolean,
```

Comment:
Is there a rule for controlling whether unused variables need a leading underscore? I'm in two minds about it...

Reply:
ESLint's rule is configurable such that those matching the specified pattern are allowed.

I originally set it to `^_`, but actually later enabled typescript's built in (stricter) unused checks at compile time and turned the eslint rule off.

TypeScript has built in support for ignoring `_` prefixed locals/parameters. I wouldn't go with another convention because this is built into typescript its self.